source/query/queries.txt

```diff
@@ -6,6 +6,14 @@ View Past Queries
 
 .. default-domain:: mongodb
 
+Starting in |compass| 1.31, you can view your favorite and saved 
+aggregation queries on the "My Queries" view once connected to 
```

Comment:
Maybe "aggregation queries" should be "queries and aggregation pipelines"? To make it clear that this view will show both?

Reply:
Good observation, I did not notice this until you mentioned it. I updated accordingly.